src/components/Pocket/TabletPocket.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Image, Text } from 'react-native';
+import PocketIcon from '../../assets/images/PocketIcon.png';
+import Button from '../common/Button';
+import strings from '../../localization/index';
```

Comment:
It isn't necesary to add the "/index" to the path, it automatically will look for that file when the path ends in a folder.